2016-06/june-29.md

```diff
@@ -0,0 +1,83 @@
+## June 29 ([discuss](https://github.com/webpack/meeting-notes/pull/2))
+
+### Attendees
+
+* [Tobias Koppers](http://github.com/sokra)
+* [Johannes Ewald](http://github.com/jhnns)
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Juho Vepsäläinen](http://github.com/bebraw)
+
+### Agenda
+
+#### Repo Management Tools
+
+* Discussed briefly using Github management tools to help:
+  * Move issues between repos:
+  * Apply Issues to multiple repos: 
+  * Track and coordinate milestones across repos:
+  
+#### Documentation
+
+* Tobias worked on an [outline](https://github.com/webpack/webpack.io/commit/18b7f14d686b76775d1b5de763423d121bc28f0b) that could serve as a foundation for webpack/webpack.io. 
+* Had one or two volunteers express interest in working on documentation however we will want to have a handful of folks working on documentation. (Because there is a lot to document).
+* Suggestions were made to create individual issues for each of the "topics or features" for webpack so that we can easily track our progress. 
+* Sean is still looking for a company who specializes in content creation etc. to help donate their efforts if possible.
+* We will start to tag issues in github that we have answered/found useful for people with the `documentation` tag so that we can address them in the future documentation
+
+#### Releasing Webpack 2.x Official
+
+* Sean has created a filter of [current reported issues](https://github.com/webpack/webpack/issues?q=is%3Aopen+is%3Aissue+label%3Abug+label%3Awebpack-2+sort%3Acreated-asc+label%3A%22in+planning%22) that Tobias is going to look through to see if it has been already fixed, or if it is by design/not a bug, etc. 
+* We agreed that a 2.x RC/Official Release milestone will be created that will assist us in tracking what remains. This will propose somewhat of a challenege because there is webpack/webpack-dev-server and other repo's to ensure are up to date to any breaking changes. 
+* Also working on PR Issues, support for problems from @gaearon and @gdi2290 (Strong representatives from the Angular and React communities).
+* In the end Documentation for [webpack/webpack.io](http://github.com/webpack/webpack.io) will be the **critical blocker**. Otherwise, only easy/small features/bugs remain.
+* Backporting the --env `function` style webpack config definition so that transition is easier for users can use the feature if desired going from webpack 1.x to webpack 2.x. 
```

Comment:
This is fantastic. This feature in webpack 2 made it so I could remove `cross-env` from my dependencies. This was actually the reason I created `cross-env` in the first place. Wonderful to see that it may not be needed even for webpack 1.